src/app/components_ngrx/typeahead-selector/typeahead-selector.component.ts

```diff
@@ -4,8 +4,7 @@ import {
   Output, ViewChild
 } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
-import { combineLatest } from 'rxjs/observable/combineLatest';
-import { of as observableOf } from 'rxjs/observable/of';
+import { combineLatest, of as ObservableOf } from 'rxjs';
```

Comment:
For consistency across codebases, I would prefer if we use named exports as is without changing the name.